### PR TITLE
Create linters.yml

### PR DESCRIPTION
**Pull request by Mohammad-Khodadadi7**
In this project, I
-  create repo
- setup linter

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (OyePriscilla) [inline .gitignore]: - [ ] Kindly create `test.md file` and add it to` .gitignore` as well according to the project requirement. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Mohammad-Khodadadi7  😃  , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Commit messages meaningful ✔️ 

## Required Changes ♻️

- [ ] Kindly ensure to customize this [Readme](https://github.com/microverseinc/readme-template) file template. This will give your repo a professional touch and be a guide to anyone that wants to access your project. Thanks. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mohammad-Khodadadi7** [comment]
Thanks for you are guidance.
pls check it again

File: .gitignore (hunk 1 above)
Comment (Abenezer-Tilahun) [reply .gitignore]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Mohammad-Khodadadi7 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Good Commit Messages


## Optional suggestions

- [ ] Kindly as suggested by the previous reviewer Kindly ensure to customize this [Readme](https://github.com/microverseinc/readme-template) file template. This will give your repo a professional touch and be a guide to anyone that wants to access your project. Thanks. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**